synapse/models/infotech.py

```diff
@@ -1224,9 +1224,15 @@ def getModelDefs(self):
                     ('created', ('time', {}), {
                         'doc': 'The time the issue was created.',
                     }),
+                    ('updated', ('time', {}), {
+                        'doc': 'The time the issue was updated.',
+                    }),
                     ('url', ('inet:url', {}), {
                         'doc': 'The URL where the issue is hosted.',
                     }),
+                    ('ext:id', ('str', {'strip': True}), {
```

Comment:
lets make this just `:id`

Reply:
The doc can probably be just `The ID of the issue in the repository system.` or something :D